### PR TITLE
Adding Nth_Ugly_Number

### LINKED ISSUE #1206: Nth Ugly number in kotlin

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1206 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding Nth_Ugly_Number in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Nth_Ugly_Number.kt

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by shubhammantri1** [comment]
Sorry @jainaman224  for creating a new PR again, But i don't know what happened with this particular branch, Everytime I add changes to it, it creates a new PR and didn't show changes in previous PR. That's why i am creating a PR again.

**Comment by shubhammantri1** [comment]
Sorry @jainaman224  for creating a new PR again, But i don't know what happened with this particular branch, Everytime I add changes to it, it creates a new PR and didn't show changes in previous PR. That's why i am creating a PR again.

File: Ugly_Number/Nth_Ugly_Number.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import java.util.*
+
+// Checking whether number is Ugly Number or not
+fun UglyNum(number: Int):Int {
+    var x = 0
```
Comment (jainaman224) [inline Ugly_Number/Nth_Ugly_Number.kt]: Fix indentation throughout the file.

File: Ugly_Number/Nth_Ugly_Number.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+       else {
+           x = -1
+           break
+       }
+   }
+   return x
+}
+
+// Finding Nth Ugly Number
+fun NthUglyNum(num: Int):Int {
```
Comment (jainaman224) [inline Ugly_Number/Nth_Ugly_Number.kt]: Space after second `:`.

File: Ugly_Number/Nth_Ugly_Number.kt (hunk 1 above)
Comment (shubhammantri1) [reply Ugly_Number/Nth_Ugly_Number.kt]: Done

File: Ugly_Number/Nth_Ugly_Number.kt (hunk 2 above)
Comment (shubhammantri1) [reply Ugly_Number/Nth_Ugly_Number.kt]: Done

File: Ugly_Number/Nth_Ugly_Number.kt (hunk 1 above)
Comment (jainaman224) [reply Ugly_Number/Nth_Ugly_Number.kt]: Not fixed. There are still 3 spaces.

**Comment by MastersAbh** [comment]
@shubhammantri1 Please fix the indentation through the file.
@rishabhgarg25699 @jainaman224 you can take the decision on this PR.

**Comment by MastersAbh** [comment]
@jainaman224 @rishabhgarg25699 The contributor is inactive. You can close the PR.

**Pull request closed without merging**